### PR TITLE
Single session for external photometry request

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR uses a single session to make external requests.

### CODE REVIEW

File: skyportal/facility_apis/atlas.py [hunk 1]
```diff
@@ -280,6 +273,7 @@ def get(request):
                         req.id,
                         instrument.id,
                         request.requester.id,
+                        session,
```
Comment (stefanv) [inline skyportal/facility_apis/atlas.py:276]: No, this one is naughty. Can't steal a session for an executor!

**Review by stefanv — COMMENTED** [review]
Looks good, other than the executor usage of session.

File: skyportal/facility_apis/atlas.py (hunk 1 above)
Comment (mcoughlin) [reply skyportal/facility_apis/atlas.py]: @stefanv makes sense. Removed.

File: skyportal/handlers/api/photometry_request.py [hunk 2]
```diff
@@ -27,10 +29,13 @@ def get(self, request_id):
                 schema: Success
         """
 
+        Session = scoped_session(
```
Comment (stefanv) [inline skyportal/handlers/api/photometry_request.py:32]: This should become a bit simpler with @guynir42's refactor.

**Review by stefanv — APPROVED** [review]

**Review by profjsb — APPROVED** [review]

**Pull request merged**